### PR TITLE
Communication Area additions

### PR DESCRIPTION
**Pull request by Theyoungster**
Re-added a Slope Jump, and added a new Boost Ball way to access the item in Communication Area.

### CODE REVIEW

**Comment by dyceron** [comment]
Did you edit the json file manually or within randovania? You're missing the changes in Temple Grounds.txt. Open randovania and save the database to update the txt file.

**Comment by Theyoungster** [comment]
Yup, you caught me red-handed. Updated, should be fine now.

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,6 +31,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added: A new experimental option, Inverted Aether. In this mode, it's the Light Aether atmosphere that is dangerous! All safe zones are moved to Light Aether, but that's not enough so it's still extremely dangerous. This mode has no logic.
 
+#### Logic Database
+
+- Added: More options to reach the item in Communication Area.
```
Comment (henriquegemignani) [inline CHANGELOG.md:36]: Please be explicit on what these alternatives were.
We use the term "pickup" in this case, not item.

File: randovania/games/prime2/json_data/Temple Grounds.txt [hunk 2]
```diff
@@ -1681,8 +1681,12 @@ Extra - in_dark_aether: False
       Trivial
   > Next to Pickup
       Any of the following:
-          Space Jump Boots or Standable Terrain (Beginner)
-          Morph Ball Bomb and Morph Ball and Bomb Jump (Beginner)
+          Space Jump Boots or Slope Jump (Beginner) or Standable Terrain (Beginner)
```
Comment (henriquegemignani) [inline randovania/games/prime2/json_data/Temple Grounds.txt:1684]: I'll leave the concept of a NSJ slope jump in beginner for someone else, but I find it suspicious.

**Review by henriquegemignani — CHANGES_REQUESTED** [review]
Could you also add videos for these new tricks?

File: randovania/games/prime2/json_data/Temple Grounds.txt (hunk 2 above)
Comment (Theyoungster) [reply randovania/games/prime2/json_data/Temple Grounds.txt]: You're entirely correct; someone else mentioned the slope jump being beginner, and I didn't actually test it before adding. I've upgraded it to Intermediate.

**Review by henriquegemignani — CHANGES_REQUESTED** [review]
That wall boost also doesn't seem very beginner.

And you haven't listed the exact new options in the changelog

**Comment by Theyoungster** [comment]
Done and done. ~~I keep forgetting wall boosts are my best trick~~

**Review by henriquegemignani — APPROVED** [review]

File: randovania/games/prime2/json_data/Temple Grounds.txt (hunk 2 above)
Comment (XenoWars) [reply randovania/games/prime2/json_data/Temple Grounds.txt]: i had mentioned it being intermediate not beginner.

**Pull request merged**